example/demo/src/app.rs

```diff
@@ -19,7 +20,7 @@ pub fn App() -> impl IntoView {
 
     view! {
         <Stylesheet id="leptos" href="/pkg/demo.css"/>
-        <HotkeysProvider>
+        <HotkeysProvider initially_active_scopes={vec!["scope_a".to_string()]}>
```

Comment:
maybe we can change initially_active_scopes to be of type `Vec<static 'str>` so we can avoid having to `.to_string()`. 

What do you think @JustBobinAround? Happy to hear other thoughts.

Reply:
I thought of doing that, I just saw you had it implemented that way. I figured there was a reason it was like that and I just didn't know what it was.

Reply:
Also, do you know if the HotkeyProvider only gets called on page load? I don't know enough about leptos to be sure and currently that Vec of scopes is being converted to a HashSet. The purpose of the HashSet kind of gets defeated if a conversion has to be made every time a key is checked.

Reply:
> Also, do you know if the HotkeyProvider only gets called on page load?
It should get called upon initial page load. (I think). 

Maybe instead of a vec of scopes, it can be a set of scopes? This would probably be better as it enforces no duplicate scopes

Reply:
> > Also, do you know if the HotkeyProvider only gets called on page load?
> > It should get called upon initial page load. (I think).
> 
> Maybe instead of a vec of scopes, it can be a set of scopes? This would probably be better as it enforces no duplicate scopes

I changed it to a set of strings and added a scopes! macro:
```rust
        <HotkeysProvider initially_active_scopes=scopes!("scope_a")>
```

I think this is good point to merge. I rework the use_hotkeys functions to a macro next pr